lib/manageiq/network_discovery/discovery.rb

```diff
@@ -7,7 +7,6 @@ module Discovery
         :ipmi            => "ManageIQ::Util::IPMI::Discovery",
         :msvirtualserver => "ManageIQ::Providers::Microsoft::Discovery",
         :mswin           => "ManageIQ::Providers::Microsoft::Discovery",
```

Comment:
Shouldn't these 2 go away also?

Reply:
Good eye.  More keywords to add to the grep.